source/adapters/CMakeLists.txt

```diff
@@ -30,31 +30,27 @@ endfunction()
 
 add_subdirectory(null)
 
-if(UR_BUILD_ADAPTER_L0 OR UR_BUILD_ADAPTER_CUDA OR UR_BUILD_ADAPTER_HIP)
-    # fetch adapter sources from SYCL
-    set(SYCL_ADAPTER_DIR "${CMAKE_CURRENT_SOURCE_DIR}/external")
-    FetchSource(https://github.com/intel/llvm.git nightly-2023-08-20 "sycl/plugins/unified_runtime/ur" ${SYCL_ADAPTER_DIR})
-endif()
-
-# Temporarily fetch the opencl adapter from a fork until the PR has been merged.
-if(UR_BUILD_ADAPTER_OPENCL)
-    # fetch adapter sources from SYCL
-    set(SYCL_ADAPTER_DIR "${CMAKE_CURRENT_SOURCE_DIR}/external")
-    FetchSource(https://github.com/fabiomestre/llvm.git opencl_adapter_unofficial "sycl/plugins/unified_runtime/ur" ${SYCL_ADAPTER_DIR})
-endif()
-
 if(UR_BUILD_ADAPTER_L0)
+    set(SYCL_ADAPTER_DIR "${CMAKE_CURRENT_BINARY_DIR}/external/level_zero")
+    FetchSource(https://github.com/intel/llvm.git nightly-2023-08-20 "sycl/plugins/unified_runtime/ur" ${SYCL_ADAPTER_DIR})
```

Comment:
Might be useful to have a variable for `nightly-2023-08-20`.

Reply:
Done.